app/sections/view-authority-asserted-integrations/macro.njk

```diff
@@ -19,7 +19,7 @@
           }) + viewDataText({
             dataTestId: 'view-data-text-additional-information',
             data: 'To find out more about these NHS assured integrations, follow the links below.',
-            classes: 'bcc-t-font-color-grey1'
+            classes: 'bcc-t-font-color--grey1'
```

Comment:
also it needs to be `bc-` rather than `bcc-`. 
`bcc-` is just for the display of the interactive library components (ie only when the library is running). If it is to be exported with the components it is just `bc-`.

Reply:
Sorted